translations/README-uz.md

```diff
@@ -304,74 +304,75 @@ Tog'ri tanlov haqida ko'proq o'qishingiz uchun manbalar:
 
 Quyida berilgan kitoblar ro'yxatida o'zim o'rganayotgan C, C++, va Python ga oid manbalarni ko'rishingiz mumkin.
 
-## Book List
+## Kitoblar ro'yxati
 
-This is a shorter list than what I used. This is abbreviated to save you time.
+Bu sizning vaqtingizni tejash uchun men ishlatgan kitoblar ro'yxatining qisqartirilgan ko'rinishi.
 
-### Interview Prep
+### Intervyuga tayyorlanish
 
 - [ ] [Programming Interviews Exposed: Coding Your Way Through the Interview, 4th Edition](https://www.amazon.com/Programming-Interviews-Exposed-Through-Interview/dp/111941847X/)
-    - answers in C++ and Java
-    - this is a good warm-up for Cracking the Coding Interview
-    - not too difficult, most problems may be easier than what you'll see in an interview (from what I've read)
+    - C++ va Java dasturlash tillarida javoblar
+    - Coding intervyuga tayyorlanish uchun juda yaxshi manba 
```

Comment:
"Cracking the Coding Interview" kitobidan oldin o'qishni maslaxat beriladi

Reply:
fixed